src/commands/python_configure_venv.yml

```diff
@@ -76,9 +76,13 @@ steps:
               - virtualenv-{{ .Environment.CACHE_VERSION }}-
         - run:
             name: Install Python packages
-            command: uv sync --frozen --all-extras << parameters.install_args >>
+            command: |
+              uv sync --frozen --all-extras << parameters.install_args >>
+              uv cache prune --ci
+            environment:
+              UV_CACHE_DIR: /tmp/.uv-cache
```

Comment:
🐼 

To enforce parity between local dev and deployment pipeline, it would make a lot of sense to add such a env variable directly onto our base CircleCI image.

Moreover, to avoid access right issues into containers, it would be advisable to keep `uv` cache into directories owned by the user running the CI/CD container (eg preferably `./.uv-cache`).

Reply:
Good point. I'll add the `UV_CACHE_DIR=/home/circleci/.uv-cache` env variable in our base CircleCI image so that we can just do: 

```
        - save_cache:
            key: virtualenv-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum "poetry.lock" }}
            paths:
              - {{ .Environment.UV_CACHE_DIR }}
```

Reply:
I would keep it loose similarly to `pipenv` and `poetry` and refer only to the active directory (user is image specific).

Reply:
Not sure about the syntax `$ENV_VAR` vs `.Environment.ENV_VAR`. I guess `.Environment.ENV_VAR` is only for variables defined via the project settings page on CircleCI platform, right? Haven't found the corresponding documentation.

Reply:
Indeed, I was thinking about this templating ; `.Environment.ENV_VAR` probably reserved for secrets defined in CircleCI web UI while shell variable accessible through the usual syntax `$ENV_VAR`.